BullsEye/ViewController.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  ViewController.swift
+//  BullsEye
+//
+//  Created by Tim Lauter on 10/31/16.
+//  Copyright © 2016 timlauter. All rights reserved.
+//
+
+import UIKit
+import QuartzCore
+
+class ViewController: UIViewController {
+    @IBOutlet weak var slider: UISlider!
+    @IBOutlet weak var targetLabel: UILabel!
+    @IBOutlet weak var scoreLabel: UILabel!
+    @IBOutlet weak var roundLabel: UILabel!
+    
+    @IBAction func startOver() {
+        startNewGame()
+        updateLabels()
+        
+        let transition = CATransition()
+        transition.type = kCATransitionFade
+        transition.duration = 0.4
+        transition.timingFunction = CAMediaTimingFunction(name: kCAMediaTimingFunctionEaseOut)
+        view.layer.add(transition, forKey: nil)
+    }
+    
+    var currentValue: Int = 0
+    var targetValue: Int = 0
+    var score: Int = 0
+    var round: Int = 0
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        
+        // let thumbImageNormal = UIImage(named: "SliderThumb-Normal")!
+        let thumbImageNormal = #imageLiteral(resourceName: "SliderThumb-Normal")
+        slider.setThumbImage(thumbImageNormal, for: .normal)
+        
+        let thumbImageHighlighted = UIImage(named: "SliderThumb-Highlighted")!
+        slider.setThumbImage(thumbImageHighlighted, for: .highlighted)
+        
+        let insets = UIEdgeInsets(top: 0, left: 14, bottom: 0, right: 14)
+        
+        let trackLeftImage = UIImage(named: "SliderTrackLeft")!
+        let trackLeftResizable = trackLeftImage.resizableImage(withCapInsets: insets)
+        
+        slider.setMinimumTrackImage(trackLeftResizable, for: .normal)
+        
+        let trackRightImage = UIImage(named: "SliderTrackRight")!
+        let trackRightResizable = trackRightImage.resizableImage(withCapInsets: insets)
+        
+        slider.setMaximumTrackImage(trackRightResizable, for: .normal)
+        
+        
+        startNewGame()
+        updateLabels()
+    }
+
+    override func didReceiveMemoryWarning() {
```

Comment:
remove this unless you are doing something when iOS tells the app its running out of memory